addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -60,113 +58,258 @@
+                            attrs="{'invisible': ['|',('mailing_type', '!=', 'mail'),('state', 'in', ('draft','test'))]}"
+                            class="oe_stat_button">
+                            <field name="opened_ratio" string="Opened" widget="percentpie"/>
+                        </button>
+                        <button name="action_view_clicked"
+                            type="object"
+                            context="{'search_default_filter_clicked': True}"
+                            attrs="{'invisible': [('state', 'in', ('draft','test'))]}"
+                            class="oe_stat_button">
+                            <field name="clicks_ratio" string="Clicked" widget="percentpie"/>
+                        </button>
+                        <button name="action_view_replied"
+                            type="object"
+                            context="{'search_default_filter_replied': True}"
+                            attrs="{'invisible': ['|',('mailing_type', '!=', 'mail'),('state', 'in', ('draft','test'))]}"
+                            class="oe_stat_button">
+                            <field name="replied_ratio" string="Replied" widget="percentpie"/>
+                        </button>
+                        <button name="action_view_bounced"
+                            type="object"
+                            context="{'search_default_filter_bounced': True}"
+                            attrs="{'invisible': [('state', 'in', ('draft','test'))]}"
+                            class="oe_stat_button">
+                            <field name="bounced_ratio" string="Bounced" widget="percentpie"/>
+                        </button>
+                    </div>
+                    <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
+                    <group class="o_mass_mailing_mailing_group">
+                        <field name="active" invisible="1"/>
+                        <field name="mailing_type" widget="radio" options="{'horizontal': true}" invisible="1"
+                            attrs="{'readonly': [('state', '!=', 'draft')]}" force_save="1"/>
+                        <field class="o_text_overflow" name="subject" string="Subject" attrs="{'invisible': [('mailing_type', '!=', 'mail')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'mail')]}" widget="char_emojis" placeholder="e.g. New Sale on all T-shirts"
+                            help="For an Email, Subject your Recipients will see in their inbox.
+                                For an SMS Text Message, internal Title of the Message."/>
+                        <field class="o_text_overflow" name="sms_subject" string="Title" placeholder="e.g. Black Friday SMS coupon" attrs="{'invisible': [('mailing_type', '!=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'sms')]}"/>
```

Comment:
Could you ensure you always give "name" first ? That the most important think when reading a view, having <field name="..." /> as other parts of there only if you want to go deeper.